### PR TITLE
Fix Scroll under add collaborators

### PR DESCRIPTION
**Pull request by andyxheli**
Fix for https://github.com/nextcloud/photos/issues/1271

@artonge please review

Signed-off-by: Andy Xheli <axheli@axtsolutions.com>

### CODE REVIEW

**Review by artonge — APPROVED** [review]

File: src/components/Albums/CollaboratorsSelectionForm.vue [hunk 1]
```diff
@@ -351,7 +351,9 @@ export default {
 
 		&__list {
 			padding: 8px;
-
+			height: 500px;
+			overflow: scroll;
+
```
Comment (artonge) [inline src/components/Albums/CollaboratorsSelectionForm.vue:356]: You need to remove the spaces here

**Comment by artonge** [comment]
Hey @andyxheli could you compile and squash ? I can't do it as your branch is on your repository

**Comment by andyxheli** [comment]
> Hey @andyxheli could you compile and squash ? I can't do it as your branch is on your repository

Hey! I think i did it. If not can you please let me know how ?

**Comment by artonge** [comment]
> Hey! I think i did it. If not can you please let me know how ?

```
npm ci
npm run build
git add .
git commit --amend
git push --force
```

**Comment by andyxheli** [comment]
Having an issue running 

axheli@Thinkpad-T470s:~/photos$ sudo npm run build

> photos@1.8.0 build /home/axheli/photos
> NODE_ENV=production webpack --progress --config webpack.js

[webpack-cli] Failed to load '/home/axheli/photos/webpack.js' config
[webpack-cli] { Error: Cannot find module 'vue-loader'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/home/axheli/photos/node_modules/@nextcloud/webpack-vue-config/webpack.config.js:26:29)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3) code: 'MODULE_NOT_FOUND' }
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! photos@1.8.0 build: `NODE_ENV=production webpack --progress --config webpack.js`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the photos@1.8.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2022-10-05T19_49_52_793Z-debug.log

**Comment by andyxheli** [comment]
Running 
npm -v
8.5.1

**Comment by andyxheli** [comment]
I dont think i have permissions  

axheli@Thinkpad-T470s:~/photos$ git push --force
remote: Permission to nextcloud/photos.git denied to andyxheli.
fatal: unable to access 'https://github.com/nextcloud/photos.git/': The requested URL returned error: 403

**Comment by andyxheli** [comment]
@artonge I got it! Thank you for your support :) Please review

**Comment by artonge** [comment]
Great ! Could you tell me what was blocking ? Just to know.

**Comment by andyxheli** [comment]
> Great ! Could you tell me what was blocking ? Just to know.

All I did was delete the photos folder from my computer then i when into my ford redownload it then did a 

gh pr checkout 1312
npm ci
npm run build
git add .
git commit --amend
git push --force

and i didnt get the error. Maybe i was doing something wrong ?

**Pull request merged**